### PR TITLE
Fix national outputting list for addr_full

### PR DESCRIPTION
**Pull request by jimbofreedman**

### CODE REVIEW

File: locations/spiders/national.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def parse(self, response):
                 "name": loc["name"],
                 "brand": loc["brand"],
                 "phone": loc["phones"][0]["phone_number"],
-                "addr_full": loc["address"]["street_addresses"],
+                "addr_full": ", ".join(loc["address"]["street_addresses"]),
```
Comment (Cj-Malone) [inline locations/spiders/national.py:40]:
```suggestion
                "street_address": ", ".join(loc["address"]["street_addresses"]),
```

**Comment by jimbofreedman** [comment]
> Thank you. [map](https://data.alltheplaces.xyz/map.html?show=https://data.alltheplaces.xyz/ci/5736937306/national/output.geojson)
> 
> It looks like this is just the small part of the address, so it should really go in `street_address`.

Now you mention it - yes!

**Review by Cj-Malone — APPROVED** [review]

**Comment by jimbofreedman** [comment]
I can't merge but I'm 👍

**Comment by iandees** [comment]
Thanks for the fix!

**Pull request merged**